### ISSUE TITLE
Cannot tab to editor, simulator in share project `embed` screen

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2329]
**Describe the bug**

In the share project screen, after sharing project, cannot tab to 

**To Reproduce**
1. Go to editor
2. Click on `share`, then `publish`
3. click on `embed` at the bottom
4. try to tab through, see that you cannot reach ``Editor`` or ``Simulator`` embed

**Expected behavior**

should be able to toggle through different embed options

**Screenshots**

![2019-08-02 14 25 43](https://user-images.githubusercontent.com/5615930/62399782-dc909a80-b531-11e9-9076-b3f283952cd1.gif)
